app/lib/donations/controllers/DonorsChooseDonationController.js

```diff
@@ -187,11 +190,11 @@ DonorsChooseDonationController.prototype.findProject = function(mobileNumber, co
       // their email address and attempts to find the document to be updated. 
       donationModel.create(currentDonationInfo).then(function(doc) {
         logger.log('debug', 'DonorsChoose.findProject created donation_infos document:%s for user:%s', JSON.stringify(doc), mobileNumber);
-        mobilecommons.profile_update(mobileNumber, config.start_donation_flow, mobileCommonsCustomFields); // Arguments: phone, optInPathId, customFields.
+        mobilecommons.profile_update(mobileNumber, donorsChooseConfig.oip_ask_first_name, mobileCommonsCustomFields);
       }, promiseErrorCallback('DonorsChoose.findProject unable to create donation_infos document for user: ' + mobileNumber));
     }
     else {
-      sendSMS(mobileNumber, config.error_direct_user_to_restart);
```

Comment:
Same here - `error_direct_user_to_restart ` property was removed in the 2015 schema and would have caused an error.